Sources/Twift.swift

```diff
@@ -5,6 +5,7 @@ import Combine
 public class Twift: NSObject, ObservableObject {
   /// The type of authentication access for this Twift instance
   public private(set) var authenticationType: AuthenticationType
+  public private(set) var oauthUser: OAuth2User?
```

Comment:
I’d suggest making this a computed property so we don’t need to worry about it getting out of sync with the `authenticationType` value:

```suggestion
  public var oauthUser: OAuth2User? {
    switch authenticationType {
      case .oauth2UserAuth(let user):
        return user
      default:
        return nil
    }
  }
```

Reply:
ahh nice! Thought it would be nice to do it in a similar fashion was not sure how to set it up 👍